x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/anomalies/anomalies.tsx

```diff
@@ -33,6 +37,16 @@ export const Anomalies = () => {
         hideDatePicker
         request$={request$}
       />
+      <div style={{ textAlign: 'right' }}>
+        <FeatureFeedbackButton
+          data-test-subj="infraMLHostFlyoutFeedbackLink"
+          formUrl={INFRA_ML_FLYOUT_FEEDBACK_LINK}
+          kibanaVersion={kibanaVersion}
+          isCloudEnv={isCloudEnv}
+          isServerlessEnv={isServerlessEnv}
+          nodeType="host"
```

Comment:
Here you can use `asset.type` to make sure that you always set the correct type based on the asset. Currently, we have only `host` type but using the asset type will give us more flexibility in the future
```suggestion
          nodeType={asset.type}
```

Reply:
`FeatureFeedbackButton` component is already limited to 'host' or 'pod' should we also extend that `nodeType` prop type there?

Reply:
I checked the form and we can check if we have `asset.type === 'pod'` and return `pod` otherwise return `host` but as we currently don't support pods in the host view we can leave it for now 👍